Python/ceval.c

```diff
@@ -1978,6 +1980,7 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, InterpreterFrame *frame, int thr
             if (sub == NULL) {
                 goto error;
             }
+            next_instr += _PyOpcode_InlineCacheEntries[BINARY_OP];
```

Comment:
We want to use a macro here. `next_instr += BINARY_OP_CACHE_SIZE`

The reason for this is that we want to define the caches as structs, like [this](https://github.com/faster-cpython/cpython/blob/a20963469638b76f451899bef634fa2e8161fef2/Include/internal/pycore_code.h#L63)

Then we define the `SIZE` macro using `sizeof` for consistency, like [this](https://github.com/faster-cpython/cpython/blob/a20963469638b76f451899bef634fa2e8161fef2/Include/internal/pycore_code.h#L72).

We can then ensure consistency between the struct def and `opcode.py` with an assert:
`assert(BINARY_OP_CACHE_SIZE == _PyOpcode_InlineCacheEntries[BINARY_OP]);`